dev-python/pyside/pyside-9999.ebuild

```diff
@@ -135,8 +142,27 @@ src_configure() {
 
 	configuration() {
 		local mycmakeargs=(
+			-DPYTHON_EXECUTABLE="${PYTHON}"
```

Comment:
I have a patch locally that adds this. I was going to push it tomorrow after https://codereview.qt-project.org/#/c/193590/ is merged upstream. Without that upstream change, passing *only* `-DPYTHON_EXECUTABLE` to cmake breaks the configure phase for me.